Monika After Story/game/script-topics.rpy

```diff
@@ -15413,3 +15413,49 @@ label monika_why_do_you_read:
     extend 3eua "While my situation in here has its downsides, [line_mid] opportunity to spend more time on the things I like."
     m 1ekbsu "...Though then again, nothing could ever beat spending more time with you~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_tabletop_rping",
+            category=['monika','literature'],
```

Comment:
I feel this should go in the 'game' category only.

Reply:
Maybe both? It is a form of literature after all. Yes, not all forms of TRPGs are written down exactly verbatim, but whether it's spoken or written, it's an expression of language through which thoughts are conveyed.

Reply:
I would have to agree with putting it in the 'literature' category as well 'game'.
Tabletop RPGs, and especially D&D, found their roots in literature and fantasy novels, and writing is usually a core component of the experience (Someone has to setup the universe in which the players/heroes will evolve, and which obstacles will stand on their way. Not to mention each player character's potential backstory, personality, and so on).

Reply:
I'll add it in both for now, and we'll discuss it after we see the new draft.